activemodel/lib/active_model/locale/en.yml

```diff
@@ -14,6 +14,7 @@ en:
       empty: "can't be empty"
       blank: "can't be blank"
       present: "must be blank"
+      required: "must exist"
```

Comment:
Sure, I'll handle it tomorrow. :+1:

Carlos Antonio da Silva - via celular
On Feb 1, 2015 12:55 AM, "Aaron Patterson" <notifications@github.com> wrote:

> In activemodel/lib/active_model/locale/en.yml
> <https://github.com/rails/rails/pull/18700#discussion_r23894297>:
>
> > @@ -14,6 +14,7 @@ en:
> >        empty: "can't be empty"
> >        blank: "can't be blank"
> >        present: "must be blank"
> > +      required: "must exist"
>
> @carlosantoniodasilva <https://github.com/carlosantoniodasilva> can you
> add it? [image: :pray:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/18700/files#r23894297>.
>